test/system/zephyr/threads.c

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2016, Xilinx Inc. and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <metal/sys.h>
+#include <metal/utilities.h>
+#include "metal-test.h"
+
+int metal_test_run(int threads, metal_thread_t child, void *arg)
+{
+	metal_unused(threads);
+	metal_unused(child);
+	metal_unused(arg);
+	metal_err("metal_test_run is currently not implemented for Zephyr\n");
```

Comment:
Seems that metal_err macro does not accept simple message without parameter... this is another topic. Not mandatory to fix it for this PR.
you probably can fix the warning using
`metal_err("%s is currently not implemented for Zephyr\n", __func__);`